Failure on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.42663&attempt_id=fdca5271-064c-4234-945d-32f5e1b817a8)

Attempted: firebird-emu

<details><summary>Partial log (click to expand)</summary><p>

```
<artificial>:(.text+0x9fb4): relocation truncated to fit: R_AARCH64_LDST64_ABS_LO12_NC against symbol `translation_sp' defined in .data section in asmcode_aarch64.o
<artificial>:(.text+0x9fb4): warning: One possible cause of this error is that the symbol is being referenced in the indicated code as if it had a larger alignment than was declared where it was defined.
<artificial>:(.text+0xa000): relocation truncated to fit: R_AARCH64_LDST64_ABS_LO12_NC against symbol `translation_sp' defined in .data section in asmcode_aarch64.o
<artificial>:(.text+0xa000): warning: One possible cause of this error is that the symbol is being referenced in the indicated code as if it had a larger alignment than was declared where it was defined.
<artificial>:(.text+0xa078): relocation truncated to fit: R_AARCH64_LDST64_ABS_LO12_NC against symbol `translation_sp' defined in .data section in asmcode_aarch64.o
<artificial>:(.text+0xa078): warning: One possible cause of this error is that the symbol is being referenced in the indicated code as if it had a larger alignment than was declared where it was defined.
collect2: error: ld returned 1 exit status
make: *** [Makefile:338: firebird-emu] Error 1
builder for '/nix/store/iab872rn7kpapp6hk8i2kgvfmznl24rd-firebird-emu-1.4.drv' failed with exit code 2
error: build of '/nix/store/iab872rn7kpapp6hk8i2kgvfmznl24rd-firebird-emu-1.4.drv' failed
```
</p></details>

--- a/pkgs/misc/emulators/firebird-emu/default.nix
+++ b/pkgs/misc/emulators/firebird-emu/default.nix
@@ -32,6 +32,7 @@ stdenv.mkDerivation rec {
     description = "Third-party multi-platform emulator of the ARM-based TI-Nspire™ calculators";
     license = stdenv.lib.licenses.gpl3;
     maintainers = with stdenv.lib.maintainers; [ pneumaticat ];
-    platforms = stdenv.lib.platforms.unix;
+    # Only tested on Linux, but likely possible to build on, e.g. macOS
+    platforms = stdenv.lib.platforms.linux;
   };
 }

